VideoLocker/assets/config/profiles.json

```diff
@@ -0,0 +1,56 @@
+{
+    "fields":[
+    {
+              "name" : "account_privacy",
+              "label" : "edX learners can see my:",
+              "type" : "switch",
+              "options" : {
+                "values" : [
+                            { "name" : "Full Profile", "value" : "all_users" },
+                            { "name" : "Limited Profile",  "value" : "private" }
+                 ]
+              },
+              "instructions" : "A limited profile only shares your username, though you can still choose to add your own profile photo.",
```

Comment:
I think you're right. At least, the API still exposes the profile image even if you're profile is limited.

So, I'll change this to "A limited profile only shares your username and profile photo." Unless I hear otherwise.